qiime/plugin/plugin.py

```diff
@@ -188,19 +188,19 @@ class PluginMethods(PluginActions):
     # TODO is `register` a better name now that functions are the only accepted
     # source (i.e. markdown support is gone)?
     def register_function(self, function, inputs, parameters, outputs, name,
-                          description, help_text=None):
+                          description, help_texts=None):
```

Comment:
Any thoughts on scraping doc strings of the function instead?


```
from numpydoc.docscrape import FunctionDoc
doc = FunctionDoc(function)
print(doc["Parameters"])

>>> [('a', 'int, default: 5', ['Does something cool']), ('b', 'str', ['Wow'])]
```

Once we make the callable this info is gone, but as it is being pulled in during registration it is all there.